tests/rules/data/syntax-error-in-prepared-statement.php

```diff
@@ -318,4 +318,9 @@ public function errorOnQueryWithoutArgs(\Doctrine\DBAL\Connection $connection)
         $query = 'SELECT email adaid gesperrt freigabe1u1 FROM ada';
         $connection->executeQuery($query);
     }
+
+    public function preparedNamedParamsSubstitution(Connection $connection)
```

Comment:
I would prefer a separate test file named `bug361.php` So the existing test files will not grow too huge